editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1018,18 +1018,19 @@ void SpriteFramesEditor::_update_library(bool p_skip_selector) {
 	}
 
 	for (int i = 0; i < frames->get_frame_count(edited_anim); i++) {
-		String name;
+		String name = itos(i);
 		Ref<Texture2D> texture = frames->get_frame_texture(edited_anim, i);
 		float duration = frames->get_frame_duration(edited_anim, i);
-		String duration_string;
-		if (duration != 1.0f) {
-			duration_string = String::utf8(" [ ×") + String::num_real(frames->get_frame_duration(edited_anim, i)) + " ]";
-		}
 
 		if (texture.is_null()) {
-			name = itos(i) + ": " + TTR("(empty)") + duration_string;
-		} else {
-			name = itos(i) + ": " + texture->get_name() + duration_string;
+			texture = empty_icon;
```

Comment:
Perhaps this should be fixed in the `ItemList` instead of the `empty_icon` hack?